discover/deployment/articles/01-installing-search-engine/01-configuring-elasticsearch/03-logging.markdown

```diff
@@ -0,0 +1,124 @@
+    these contents: 
+
+        appender.console.layout.pattern=[%d{ISO8601}][%-5p][%-25c{1.}] %marker%m%n
+        appender.console.layout.type=PatternLayout
+        appender.console.name=console
+        appender.console.type=Console
+        logger.action.level=info
+        logger.action.name=org.elasticsearch.action
+        rootLogger.appenderRef.console.ref=console
+        rootLogger.level=info
+        status=error
+
+    Read the Elasticsearch 
+    [logging documentation](https://www.elastic.co/guide/en/elasticsearch/reference/6.x/logging.html) 
+    for more information.
+
+4.  Restart the @product@ server, or just the embedded Elasticsearch.
+
+5.  To restart just the embedded Elasticsearch server, create a file called
+
+        com.liferay.portal.bundle.blacklist.internal.BundleBlacklistConfiguration.config 
+
+    in `Liferay Home/osgi/configs` directory and populate it with
+
+        blacklistBundleSymbolicNames="com.liferay.portal.search.elasticsearch6.impl"
+
+6.  Save the `.config` file and the bundle is stopped. Once you see the
+    `[STOPPED]` message in the logs, restart the bundle by commenting the line
+    out and re-saving the file:
+
+        #blacklistBundleSymbolicNames="com.liferay.portal.search.elasticsearch6.impl"
+
+Once this is accomplished, the embedded Elasticsearch server is properly set up
+to display Elasticsearch logs at the INFO level and above.
+
+<!-- Do we want to say a little more about what we're doing here? -->
+
+Two common Elasticsearch logs can be configured further in @product@: the Slow
+Log and the JVM's Garbage Collection log.
+
+## Configuring Slow Log
+
+Read about Elasticsearch's Slow Log
+[here](https://www.elastic.co/guide/en/elasticsearch/reference/6.1/index-modules-slowlog.html). 
+
+Configure the Slow Log for the embedded Elasticsearch server in the
+Elasticsearch 6 entry in System Settings. Add these properties to the
+Additional Index Configurations property:
+
+
+    index.indexing.slowlog.threshold.index.debug: 2s
+    index.indexing.slowlog.threshold.index.info: 5s
+    index.indexing.slowlog.threshold.index.trace: 500ms
+    index.indexing.slowlog.threshold.index.warn: 10s
+
+    index.search.slowlog.threshold.fetch.debug: 500ms
+    index.search.slowlog.threshold.fetch.info: 800ms
+    index.search.slowlog.threshold.fetch.trace: 200ms
+    index.search.slowlog.threshold.fetch.warn: 1s
+
+    index.search.slowlog.threshold.query.debug: 2s
+    index.search.slowlog.threshold.query.info: 5s
+    index.search.slowlog.threshold.query.trace: 500ms
+    index.search.slowlog.threshold.query.warn: 10s
+
+These are example values. Adjust as needed.
+
+## Configuring JVM Garbage Collection Logging
+
+As with the Slow Log configuration, Elasticsearch's JVM Garbage Collection
+logging is adjustable in the Elasticsearch 6 entry in System Settings. Add these
+properties to the Additional Index Configurations property:
```

Comment:
Per LPS-85124 and my testing, the properties should be added to "Addtional Configurations" not "Additional Index Configurations"

Reply:
so although LPS-85124 was mainly about reducing the Embedded logging, i dont think the documentation really needs to cover anything for embedded because its not a supported mode, and it doesnt really make sense for a customer to monitor gc or performance when in embedded mode. 

but LPS-85124 did have an effect on remote mode as well, which is why we needed to write documentation on it. in remote mode the settings would need to be under "Additional Index Configurations"

Reply:
okay thanks for the clarification!

Reply:
ya embedded settings go under "Additional Configurations"

Reply:
Good catch!

Reply:
@BryanEngler @joshchong can we think of a quick and easy way to demonstrate that the settings are working? How do I trigger the slowlog logging or the garbage collection logging for Elasticsearch?